EasyClangComplete.sublime-settings

```diff
@@ -83,4 +83,6 @@
   // Remove cache data older than specified time.
   // Format: <hours>:<minutes>:<seconds>: "HH:MM:SS".
   "max_cache_age": "00:30:00",
+
+  "show_type_info": false
```

Comment:
In the end, when this works reliably, this should probably be enabled by default.